lib/auth/aws_certs.go

```diff
@@ -560,5 +560,118 @@ MNvXDi/ErzgrHGSpcvmGHiOhMf3UzChMWbIr6udoDlMbSIO7+8F+jUJkh4Xl1lKb
 YeN5fsLZp7T/6YvbFSPpmbn1YoE2vKtuGKxObRrhU3h4JHdp1Zel1pZ6lh5iM0ec
 SD11SximGIYCjfZpRqI3q50mbxCd7ckULz+UUPwLrfOds4VrVVSj+x0ZdY19Plv2
 9shw5ez6Cn7E3IfzqNHO
+-----END CERTIFICATE-----`),
+	"ap-south-2": []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
These are the RSA certificates, not the RSA 2048 ones.

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/regions-certs.html

There are three tabs for every region -- DSA, RSA, and RSA 2048. We need to use the RSA 2048 ones.

Reply:
My bad. Fixed in [f2ea30a](https://github.com/gravitational/teleport/pull/39045/commits/f2ea30a90618a1609927e56b626e75c5553be49c)